risc0/zkvm/src/exec/tests.rs

```diff
@@ -666,7 +666,7 @@ fn memory_access() {
         Executor::from_elf(env, MULTI_TEST_ELF).unwrap().run()
     }
 
-    access_memory(0x0000_0000).err().unwrap();
-    access_memory(0x0C00_0000).err().unwrap();
+    access_memory(0x0000_0000).unwrap();
+    access_memory(0x0C00_0000).unwrap();
```

Comment:
I think we still need to test for the condition that we expect a fault.

Reply:
yeah. I updated the test to use a helper to expect a fault or not. I wonder if I should add functions to the `Session` and `Receipt`? It kind of looks clunky